neutrino.go

```diff
@@ -1251,16 +1251,14 @@ func (s *ChainService) handleDonePeerMsg(state *peerState, sp *ServerPeer) {
 		list = state.outboundPeers
 	}
 	if _, ok := list[sp.ID()]; ok {
-		if !sp.Inbound() && sp.VersionKnown() {
```

Comment:
where do we ensure no peer can make inbound connections to us?

Reply:
I don't think we ever listen using the `connmgr`?

Reply:
> I don't think we ever listen using the connmgr?

Correct. There's no logic to even handle inbound connections at the moment.